docs/sources/flow/reference/components/prometheus.scrape.md

```diff
@@ -54,38 +54,40 @@ Name | Type | Description | Default | Required
 `label_limit`              | `uint`     | More than this many labels post metric-relabeling causes the scrape to fail. | | no
 `label_name_length_limit`  | `uint`     | More than this label name length post metric-relabeling causes the scrape to fail. | | no
 `label_value_length_limit` | `uint`     | More than this label value length post metric-relabeling causes the scrape to fail. | | no
+`bearer_token`             | `secret`   | Bearer token to authenticate with. | | no
```

Comment:
copy pasting this part of the table for http_client_config on every doc that is is squashed seems like a maintenance headache. Any thoughts on an alternative?

Reply:
What about a new shared doc which holds the common rows in the table when the HTTP client config is squashed?